docs/tutorials/blog.rst

```diff
@@ -287,12 +287,12 @@ page when the URL is set to ``/my_blog/all``. So, your new
     function my_blog_page_handler($segments) {
         switch ($segments[0]) {
             case 'add':
-               include elgg_get_plugins_path() . 'my_blog/pages/my_blog/add.php';
+               include __DIR__ . '/pages/my_blog/add.php';
                break;
 
             case 'all':
             default:
-               include elgg_get_plugins_path() . 'my_blog/pages/my_blog/all.php';
+               include __DIR__ . '/pages/my_blog/all.php';
```

Comment:
I thought we/I updated these to use elgg_view_resource... maybe that PR didn't get submitted yet...